openwisp_radius/base/models.py

```diff
@@ -1034,7 +1035,14 @@ class AbstractOrganizationRadiusSettings(UUIDModel):
         null=True,
         blank=True,
         help_text=_(
-            'Additional configuration for SMS backend in JSON format, if needed'
+            'Additional configuration for SMS backend in JSON format (optional)'
+        ),
+    )
+    freeradius_allowed_hosts = models.TextField(
+        null=True,
+        blank=True,
+        help_text=(
+            'Comma seperated list of IP addressses allowed to access freeradius API'
```

Comment:
typo: seperated > separated
typo: addressses > addresses

I would change it to:

`Comma separated list of IP addresses allowed to consume the freeradius API endpoints.`

for some reason this is not appearing in the admin, It's most probably happening because you're redefining the admin widget to indicate the number of rows and columns, but you're not passing the `help_text`, so it's getting lost.